client/README.md

```diff
@@ -23,17 +21,16 @@ This extension provides Puppet Language support for [Visual Studio Code](https:/
 - Import from `puppet resource` directly into manifests
 - Node graph preview
 
+
 ![Example of features](docs/assets/language_server.gif)
```

Comment:
This will display once we follow [these instructions](https://code.visualstudio.com/docs/extensions/publish-extension#_marketplace-integration) in the vsce package step